types/jquery.contextmenu/index.d.ts

```diff
@@ -22,8 +22,8 @@ interface JQueryContextMenuOptions {
         hide?: string;
     };
     events?: {
-        show?: () => void;
-        hide?: () => void;
+        show?: (options?: any) => any;
```

Comment:
The docs seem to imply that the return value should be boolean?

Reply:
Agreed